docs/cubeviz/import_data.rst

```diff
@@ -51,7 +51,7 @@ Importing data through the GUI
 
 You can load your data into the Cubeviz application
 by clicking the :guilabel:`Import Data` button at the top left of the application's
-user interface. This opens a dialogue where the user can select a file
+user interface. This opens a dialogue where you can select a file
```

Comment:
I would leave as much as possible the reference to "the user" and not to "you".
It would be good to check that this is consistent throughout the docs at some point.

Reply:
The documentation is very inconsistent on this choice, even looking at different sections of a single RST file. This particular section mostly used "you," which is why I made the switch. Would you rather I change this section to "the user" instead? I agree that a more widespread check is a future task.

Reply:
Please proceed with using "the user". I will make a note to update everywhere else.